docs-v1/smart-contracts/apis/ParcelNFT.md

```diff
@@ -0,0 +1,177 @@
+# ParcelNFT
+
+![Use Case Diagram for the Vault](../../../.gitbook/assets/vault-use-case.png)
+
+## External
+
+
+### `setBaseURI`
+
+```
+
+ setBaseURI(string calldata __baseURI)
+ 
+```
+
+Sets a new BaseURI. The caller must have the Roles.PARCEL_MANGER role. Implemented in `AccessControlUpgradeable` in gnus.ai/contracts-upgradeable-diamond. 
```

Comment:
Since a lot of these functions do have code comments, can we update the documentation on the contracts if it's not enough and then generate some of the docs from the code comments themselves?